solidity/test/LiquidityProtection.js

```diff
@@ -6,7 +6,7 @@ const Decimal = require('decimal.js');
 
 const { ZERO_ADDRESS, MAX_UINT256 } = constants;
 const { duration, latest } = time;
-const { ROLE_OWNER, ROLE_WHITELIST_ADMIN, ROLE_GOVERNOR, ROLE_MINTER } = roles;
+const { ROLE_OWNER, ROLE_GOVERNOR, ROLE_MINTER } = roles;
```

Comment:
Do we still need `ROLE_WHITELIST_ADMIN` in `Constants.js`?

Reply:
I missed it but removed in the next PR